src/components/Dashboard/FilingHistoryList/PendingFiling.vue

```diff
@@ -11,9 +11,9 @@
 
     <p>If this issue persists, please contact us.</p>
 
-    <contact-info class="pt-3" />
+    <ContactInfo class="mt-4" />
 
-    <div class="to-dashboard-container">
+    <div class="to-dashboard-container text-center mt-6">
```

Comment:
The top padding was previously 1 rem. Now the top margin is 1.5rem. This is in line with the latest UI designs.